### ISSUE TITLE
Stable Diffusion WebUI Forge - Classic/Neo

### ISSUE THREAD

**Issue by JJonnahJameson** [LykosAI/StabilityMatrix#1393]
### Package

As mentioned here: https://github.com/LykosAI/StabilityMatrix/issues/1241
I'm getting this error during install process of this specific package: Could not install forge-classic (System.UnauthorizedAccessException: Access to the path is denied.
needless to say, i didn't have this issue with the previous version of Stability Matrix.
Thanks for your hard work guys.

### When did the issue occur?

Installing the Package

### What GPU / hardware type are you using?

Nvidia RTX 3070 8GB + 32GB of Ram + 12400 i5

### What happened?

Could not install forge-classic (System.UnauthorizedAccessException: Access to the path is denied.

### Console output

Unpacking resources
Unpacking resources
Cloning into 'D:\SM\Data\Packages\Stable Diffusion WebUI Forge - Classic'...
Download Complete
Using Python 3.11.13 environment at: venv
Resolved 2 packages in 274ms
Prepared 1 package in 0.30ms
Installed 1 package in 10ms
 + wheel==0.45.1
Using Python 3.11.13 environment at: venv
Resolved 139 packages in 1.92s
Prepared 139 packages in 264ms
Installed 139 packages in 2.66s
 + absl-py==2.3.1
 + addict==2.4.0
 + aiohappyeyeballs==2.6.1
 + aiohttp==3.12.15
 + aiosignal==1.4.0
 + albumentations==1.4.3
 + antlr4-python3-runtime==4.9.3
 + anyio==3.7.1
 + attrs==25.3.0
 + certifi==2025.8.3
 + cffi==2.0.0
 + charset-normalizer==3.4.3
 + clean-fid==0.1.35
 + clip==1.0 (from https://github.com/openai/CLIP/archive/d50d76daa670286dd6cacf3bcd80b5e4823fc8e1.zip)
 + colorama==0.4.6
 + coloredlogs==15.0.1
 + contourpy==1.3.3
 + cssselect2==0.8.0
 + cycler==0.12.1
 + diskcache==5.6.3
 + einops==0.8.1
 + facexlib==0.3.0
 + fastapi==0.114.2
 + filelock==3.19.1
 + filterpy==1.4.5
 + flatbuffers==25.2.10
 + fonttools==4.60.0
 + frozenlist==1.7.0
 + fsspec==2025.9.0
 + ftfy==6.3.1
 + fvcore==0.1.5.post20221221
 + gitdb==4.0.12
 + gitpython==3.1.44
 + h11==0.12.0
 + httpcore==0.15.0
 + httpx==0.24.1
 + huggingface-hub==0.30.2
 + humanfriendly==10.0
 + idna==3.10
 + imageio==2.37.0
 + inflection==0.5.1
 + iopath==0.1.10
 + jax==0.7.1
 + jaxlib==0.7.1
 + jinja2==3.1.6
 + joblib==1.5.2
 + jsonmerge==1.8.0
 + jsonschema==4.25.1
 + jsonschema-specifications==2025.9.1
 + kiwisolver==1.4.9
 + kornia==0.6.7
 + lark==1.2.2
 + lazy-loader==0.4
 + lightning==2.5.1
 + lightning-utilities==0.15.2
 + llvmlite==0.45.0
 + lxml==6.0.1
 + markdown-it-py==4.0.0
 + markupsafe==3.0.2
 + matplotlib==3.10.6
 + mdurl==0.1.2
 + mediapipe==0.10.21
 + ml-dtypes==0.5.3
 + mpmath==1.3.0
 + multidict==6.6.4
 + networkx==3.5
 + numba==0.62.0
 + numpy==1.26.4
 + omegaconf==2.2.3
 + onnx==1.19.0
 + onnxruntime==1.22.1
 + open-clip-torch==1.3.0
 + opencv-contrib-python==4.11.0.86
 + opencv-python==4.8.1.78
 + opencv-python-headless==4.11.0.86
 + opt-einsum==3.4.0
 + packaging==24.2
 + piexif==1.1.3
 + pillow==11.2.1
 + pillow-heif==0.22.0
 + platformdirs==4.4.0
 + portalocker==3.2.0
 + propcache==0.3.2
 + protobuf==4.25.7
 + psutil==5.9.8
 + pycparser==2.23
 + pydantic==1.10.22
 + pygments==2.19.2
 + pyparsing==3.2.4
 + pyreadline3==3.5.4
 + python-dateutil==2.9.0.post0
 + pytorch-lightning==2.5.5
 + pywavelets==1.9.0
 + pywin32==311
 + pyyaml==6.0.2
 + referencing==0.36.2
 + regex==2025.9.18
 + reportlab==4.4.4
 + requests==2.32.5
 + resize-right==0.0.2
 + rich==13.9.4
 + rpds-py==0.27.1
 + safetensors==0.5.3
 + scikit-image==0.21.0
 + scikit-learn==1.7.2
 + scipy==1.16.2
 + sentencepiece==0.2.1
 + setuptools==80.9.0
 + six==1.17.0
 + smmap==5.0.2
 + sniffio==1.3.1
 + sounddevice==0.5.2
 + spandrel==0.4.1
 + spandrel-extra-arches==0.2.0
 + starlette==0.38.6
 + svglib==1.5.1
 + sympy==1.14.0
 + tabulate==0.9.0
 + termcolor==3.1.0
 + threadpoolctl==3.6.0
 + tifffile==2025.9.20
 + timm==1.0.15
 + tinycss2==1.4.0
 + tokenizers==0.13.3
 + torch==2.8.0
 + torchdiffeq==0.2.5
 + torchmetrics==1.8.2
 + torchsde==0.2.6
 + torchvision==0.23.0
 + tqdm==4.67.1
 + trampoline==0.1.2
 + transformers==4.30.2
 + typing-extensions==4.15.0
 + urllib3==2.5.0
 + wcwidth==0.2.13
 + webencodings==0.5.1
 + yacs==0.1.8
 + yapf==0.43.0
 + yarl==1.20.1
Using Python 3.11.13 environment at: venv
Resolved 13 packages in 1.85s
Prepared 13 packages in 85ms
Uninstalled 12 packages in 970ms
Installed 13 packages in 1.03s
 - filelock==3.19.1
 + filelock==3.13.1
 - fsspec==2025.9.0
 + fsspec==2024.6.1
 - jinja2==3.1.6
 + jinja2==3.1.4
 - markupsafe==3.0.2
 + markupsafe==2.1.5
 ~ mpmath==1.3.0
 - networkx==3.5
 + networkx==3.3
 - numpy==1.26.4
 + numpy==2.1.2
 - pillow==11.2.1
 + pillow==11.0.0
 - sympy==1.14.0
 + sympy==1.13.3
 - torch==2.8.0
 + torch==2.8.0+cu128
 + torchaudio==2.8.0+cu128
 - torchvision==0.23.0
 + torchvision==0.23.0+cu128
 - typing-extensions==4.15.0
 + typing-extensions==4.12.2
Could not install forge-classic (System.UnauthorizedAccessException: Access to the path is denied.
   at System.IO.FileSystem.MoveFile(String sourceFullPath, String destFullPath, Boolean overwrite)
   at System.IO.FileInfo.MoveTo(String destFileName, Boolean overwrite)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at StabilityMatrix.Core.Models.FileInterfaces.FilePath.MoveToAsync(FilePath destinationFile)
   at StabilityMatrix.Core.Helper.FileTransfers.MoveFileAsync(FilePath sourceFile, FilePath destinationFile, Boolean overwrite, Boolean overwriteIfHashMatches)
   at StabilityMatrix.Core.Helper.FileTransfers.MoveAllFiles(DirectoryPath sourceDir, DirectoryPath destinationDir, Boolean overwrite, Boolean overwriteIfHashMatches)
   at StabilityMatrix.Core.Helper.FileTransfers.MoveAllFilesAndDirectories(DirectoryPath sourceDir, DirectoryPath destinationDir, Boolean overwrite, Boolean overwriteIfHashMatches, Boolean deleteSymlinks)
   at StabilityMatrix.Core.Helper.SharedFolders.CreateOrUpdateLink(DirectoryPath sourceDir, DirectoryPath destinationDir, Boolean overwrite, Boolean recursiveDelete)
   at StabilityMatrix.Core.Helper.SharedFolders.UpdateLinksForPackage[T](Dictionary`2 sharedFolders, DirectoryPath modelsDirectory, DirectoryPath installDirectory, Boolean recursiveDelete)
   at StabilityMatrix.Core.Models.Packages.BaseGitPackage.SetupModelFolders(DirectoryPath installDirectory, SharedFolderMethod sharedFolderMethod)
   at StabilityMatrix.Core.Models.PackageModification.SetupModelFoldersStep.ExecuteAsync(IProgress`1 progress)
   at StabilityMatrix.Core.Models.PackageModification.PackageModificationRunner.ExecuteSteps(IEnumerable`1 steps))

### Version

2.15.0

### What Operating System are you using?

Windows

**Comment by SethG911** [comment]
Try running Stability Matrix in admin mode. That worked for me.
